app/src/main/res/layout/contact_list_entry.xml

```diff
@@ -24,15 +24,9 @@
         <TextView
             android:id="@+id/name"
             android:layout_width="match_parent"
-            android:layout_height="wrap_content"
+            android:layout_height="match_parent"
             android:text="@string/name"
-            android:textSize="@dimen/contacts_list_entry_text_name_size" />
-
-        <TextView
-            android:id="@+id/pubKeySnippet"
```

Comment:
I disagree because of three reasons:
- A contact can have multiple keys, in that case which key should you display?
- I don't think anyone really knows the public key of their friend by heard, so it won't give you any confidence you're selecting the right person.
- There is really no need for the user to see the key except for very specific/rare cases. In these cases it should not be an issue two have to perform two additional clicks (on the contact -> on the key of interest in the `ContactActivity`).